docs/content/secure/secure-api-access-with-waf.md

```diff
@@ -0,0 +1,180 @@
+
+There are two ways to implement Coraza WAF protection with Traefik:
+
+- **Native Middleware** (Traefik Hub exclusive): A high-performance native implementation available exclusively in Traefik Hub API Gateway that provides at least 23 times better performance compared to the community plugin version.
+- **Community WASM Plugin** (Open Source Traefik Proxy): Available as a [community plugin](https://plugins.traefik.io/plugins/65f2aea146079255c9ffd1ec/coraza-waf) in the Traefik Plugin Catalog for open-source Traefik Proxy users through the [coraza-http-wasm-traefik](https://github.com/jcchavezs/coraza-http-wasm-traefik) project.
+
+!!! note "Performance Advantage"
+    The native middleware implementation in Traefik Hub API Gateway delivers significantly superior performance compared to the WASM-based community plugin available for open-source Traefik Proxy. This performance boost ensures that security enforcement does not compromise application responsiveness.
+
+!!! note "Rule Compatibility"
+    The native middleware supports the Coraza rule syntax and is compatible with [OWASP Core Rule Set (CRS)](https://coreruleset.org/docs/), allowing you to leverage proven security rules maintained by the security community.
+
+## Basic WAF Protection
+
+To protect your applications with custom security rules, apply the following configuration:
+
+```yaml tab="Middleware WAF"
+apiVersion: traefik.io/v1alpha1
+kind: Middleware
+metadata:
+  name: waf-protection
+  namespace: apps
+spec:
+  plugin:
+    coraza:
+      directives:
+        - SecRuleEngine On
+        - SecRule REQUEST_URI "@streq /admin" "id:101,phase:1,t:lowercase,log,deny"
+        - SecRule ARGS "@detectSQLi" "id:102,phase:2,block,msg:'SQL Injection Attack Detected',logdata:'Matched Data: %{MATCHED_VAR} found within %{MATCHED_VAR_NAME}'"
+```
+
+```yaml tab="IngressRoute"
+apiVersion: traefik.io/v1alpha1
+kind: IngressRoute
+metadata:
+  name: protected-app
+  namespace: apps
+spec:
+  entryPoints:
+    - websecure
+  routes:
+  - match: Path(`/my-app`)
+    kind: Rule
+    services:
+    - name: whoami
+      port: 80
+    middlewares:
+    - name: waf-protection
+```
+
+```yaml tab="Service & Deployment"
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: whoami
+  namespace: apps
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: whoami
+  template:
+    metadata:
+      labels:
+        app: whoami
+    spec:
+      containers:
+      - name: whoami
+        image: traefik/whoami
+
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: whoami
+  namespace: apps
+spec:
+  ports:
+  - port: 80
+    name: whoami
+  selector:
+    app: whoami
+```
+
+## Advanced Protection with OWASP Core Rule Set
+
+To implement comprehensive protection using the OWASP Core Rule Set, which provides battle-tested rules against common attack patterns, apply the following configuration:
```

Comment:
Could you please be more accurate on what the rules you set allow doing, and the issues they prevent?